04-simulate-an-election/election.py

```diff
@@ -0,0 +1,68 @@
+# Copyright (c) 2016 Ami . All rights reserved
+
+from random import random
+A = "a"
```

Comment:
We normally don't use one letter names for variables (except counters), and anyway without the context I wouldn't know that `A` symbolises candidate A. Please change this variable to `CANDIDATE_A` and the other to `CANDIDATE_B`.